### PR TITLE
SDK v4.1

### PR DESCRIPTION
**Pull request by CleanBread**

### CODE REVIEW

File: pages/hub/apps/sdk/data-hooks/useWaveLeaderBoard.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { Callout } from 'components'
+
+Hook is used to get the [Wave Total Leaderboard](/hub/apps/guides/azuro-waves#fetch-total-leaderboard) data or [Wave Leaderboard for Specific Period](/hub/apps/guides/azuro-waves#fetch-leaderboard-for-specific-period).
+
+<Callout type="info">
+  This hook is wrapper around [getWaveLevels](/hub/apps/sdk/utils/getWaveLevels) function.
```
Comment (on47sky) [inline pages/hub/apps/sdk/data-hooks/useWaveLeaderBoard.mdx:6]: wrong hook name and link

File: pages/hub/apps/sdk/data-hooks/useWavePeriods.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { Callout } from 'components'
+
+Hook is used to get the [Wave Periods](/hub/apps/guides/azuro-waves#fetch-existing-periods-of-a-wave) data.
+
+<Callout type="info">
+  This hook is wrapper around [getWaveLevels](/hub/apps/sdk/utils/getWaveLevels) function.
```
Comment (on47sky) [inline pages/hub/apps/sdk/data-hooks/useWavePeriods.mdx:6]: same

File: pages/hub/apps/sdk/data-hooks/useWaveStats.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import { Callout } from 'components'
+
+Hook is used to get the [Wave User Stats](/hub/apps/guides/azuro-waves#get-user-stats) data.
+
+<Callout type="info">
+  This hook is wrapper around [getWaveLevels](/hub/apps/sdk/utils/getWaveLevels) function.
```
Comment (on47sky) [inline pages/hub/apps/sdk/data-hooks/useWaveStats.mdx:6]: and here =)

File: pages/hub/releases/22-sdk-v4.1.mdx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { Callout } from 'components'
+
+# 06/26 SDK v4.1
+
+Implementation of [Azuro Waves](/hub/apps/guides/azuro-waves).
+
+## New Hooks
```
Comment (on47sky) [inline pages/hub/releases/22-sdk-v4.1.mdx:7]: useNavigation?

File: pages/hub/apps/sdk/data-hooks/useWaveLeaderBoard.mdx (hunk 1 above)
Comment (CleanBread) [reply pages/hub/apps/sdk/data-hooks/useWaveLeaderBoard.mdx]: thanks
0f86ef07421d7347f86c7aca7a532616ae866ca9

File: pages/hub/apps/sdk/data-hooks/useWaveStats.mdx (hunk 3 above)
Comment (CleanBread) [reply pages/hub/apps/sdk/data-hooks/useWaveStats.mdx]: 0f86ef07421d7347f86c7aca7a532616ae866ca9

File: pages/hub/apps/sdk/data-hooks/useWavePeriods.mdx (hunk 2 above)
Comment (CleanBread) [reply pages/hub/apps/sdk/data-hooks/useWavePeriods.mdx]: 0f86ef07421d7347f86c7aca7a532616ae866ca9

File: pages/hub/releases/22-sdk-v4.1.mdx (hunk 4 above)
Comment (CleanBread) [reply pages/hub/releases/22-sdk-v4.1.mdx:7]: ab49fa87b3e912a102bb97ed53ca0dac07be0c8c

**Review by on47sky — APPROVED** [review]

File: pages/hub/apps/sdk/data-hooks/useWaveLeaderBoard.mdx [hunk 5]
```diff
@@ -0,0 +1,51 @@
+import { Callout } from 'components'
+
+Hook is used to get the [Wave Total Leaderboard](/hub/apps/guides/azuro-waves#fetch-total-leaderboard) data or [Wave Leaderboard for Specific Period](/hub/apps/guides/azuro-waves#fetch-leaderboard-for-specific-period).
+
+<Callout type="info">
+  This hook is wrapper around [getWaveLeaderBoard](/hub/apps/sdk/utils/getWaveLeaderBoard) function.
```
Comment (Avvrik) [inline pages/hub/apps/sdk/data-hooks/useWaveLeaderBoard.mdx:6]: a wrapper

File: pages/hub/apps/sdk/data-hooks/useWaveActivation.mdx [hunk 6]
```diff
@@ -0,0 +1,44 @@
+import { Callout } from 'components'
+
+Hook is used to [Activate User Levels](/hub/apps/guides/azuro-waves#activate-user-levels).
+
+<Callout type="info">
+  This hook is wrapper around [activateWave](/hub/apps/sdk/utils/activateWave) function.
```
Comment (Avvrik) [inline pages/hub/apps/sdk/data-hooks/useWaveActivation.mdx:6]: a wrapper

File: pages/hub/apps/sdk/data-hooks/useWaveLevels.mdx [hunk 7]
```diff
@@ -0,0 +1,45 @@
+import { Callout } from 'components'
+
+Hook is used to get the [Wave Levels](/hub/apps/guides/azuro-waves#get-azuro-wave-levels-data) data.
+
+<Callout type="info">
+  This hook is wrapper around [getWaveLevels](/hub/apps/sdk/utils/getWaveLevels) function.
```
Comment (Avvrik) [inline pages/hub/apps/sdk/data-hooks/useWaveLevels.mdx:6]: a wrapper

File: pages/hub/apps/sdk/data-hooks/useWavePeriods.mdx [hunk 8]
```diff
@@ -0,0 +1,45 @@
+import { Callout } from 'components'
+
+Hook is used to get the [Wave Periods](/hub/apps/guides/azuro-waves#fetch-existing-periods-of-a-wave) data.
+
+<Callout type="info">
+  This hook is wrapper around [getWavePeriods](/hub/apps/sdk/utils/getWavePeriods) function.
```
Comment (Avvrik) [inline pages/hub/apps/sdk/data-hooks/useWavePeriods.mdx:6]: a wrapper

File: pages/hub/apps/sdk/data-hooks/useWaveStats.mdx [hunk 9]
```diff
@@ -0,0 +1,60 @@
+import { Callout } from 'components'
+
+Hook is used to get the [Wave User Stats](/hub/apps/guides/azuro-waves#get-user-stats) data.
+
+<Callout type="info">
+  This hook is wrapper around [getWaveStats](/hub/apps/sdk/utils/getWaveStats) function.
```
Comment (Avvrik) [inline pages/hub/apps/sdk/data-hooks/useWaveStats.mdx:6]: a wrapper

**Pull request merged**